tests/utils/utils.go

```diff
@@ -234,7 +234,77 @@ func SendERC20Destination(
 	Expect(event.Sender).Should(Equal(crypto.PubkeyToAddress(senderKey.PublicKey)))
 	Expect(event.Amount).Should(Equal(bridgedAmount))
 
-	return receipt, event.TeleporterMessageID, event.Amount
+	return receipt, event.Amount
+}
+
+func SendERC20MultihopAndVerify(
+	ctx context.Context,
+	network interfaces.Network,
+	fundedKey *ecdsa.PrivateKey,
+	recipientKey *ecdsa.PrivateKey,
+	recipientAddress common.Address,
+	fromSubnet interfaces.SubnetTestInfo,
+	fromBridge *erc20destination.ERC20Destination,
+	fromBridgeAddress common.Address,
+	toSubnet interfaces.SubnetTestInfo,
+	toBridge *erc20destination.ERC20Destination,
+	toBridgeAddress common.Address,
+	cChainInfo interfaces.SubnetTestInfo,
+	bridgedAmount *big.Int,
+) {
+	teleporterUtils.SendNativeTransfer(
+		ctx,
+		fromSubnet,
+		fundedKey,
+		recipientAddress,
+		big.NewInt(1e18),
+	)
+	input := erc20destination.SendTokensInput{
+		DestinationBlockchainID:  toSubnet.BlockchainID,
+		DestinationBridgeAddress: toBridgeAddress,
+		Recipient:                recipientAddress,
+		PrimaryFee:               big.NewInt(0),
+		SecondaryFee:             big.NewInt(0),
+		AllowedRelayerAddresses:  []common.Address{},
+	}
+
+	receipt, bridgedAmount := SendERC20Destination(
+		ctx,
+		fromSubnet,
+		fromBridge,
+		fromBridgeAddress,
+		input,
+		bridgedAmount,
+		recipientKey,
+	)
+
+	receipt = network.RelayMessage(
+		ctx,
+		receipt,
+		fromSubnet,
+		cChainInfo,
+		true,
+	)
+
+	receipt = network.RelayMessage(
+		ctx,
+		receipt,
+		cChainInfo,
+		toSubnet,
+		true,
+	)
```

Comment:
nit: Could you add some small comments to each step in this (and maybe other) helper function(s)? It took me a second to see that this wasn't the same call copy-pasted.